assets/js/kelseyscript.js

```diff
@@ -72,16 +91,15 @@ function printQuakesBySearch(event) {
     }
 }
 
-//This sets the formatting for the map for either function, but centers the map on the US once it loads.
-var map;
-var infowindow;
-function initMap() {
-    map = new google.maps.Map(document.getElementById('map'), {
-        zoom: 3.6,
-        center: new google.maps.LatLng(39.8283, -99.5795),
-        mapTypeId: 'terrain'
-    });
-    infowindow = new google.maps.InfoWindow();
-}
-
 
+//This is to remove the markers for the previously searched city and last 5 rumbles.
+function removeMarker() {
```

Comment:
If you reinitialize the map, removing the markers is redundant since the new map does not contain any markers.
See this change as an example: #111.

Reply:
Hi, The remove marker is also in case they click the button to restart. 

Otherwise the blue markers stay, when the map reinitializes.